src/eda_functions.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+import pandas as pd
+from sklearn.metrics import confusion_matrix
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+# set the plot style
+sns.set_theme(context='notebook', style='darkgrid', palette='crest', font='helvetica')
+cmap = sns.color_palette('crest')
+sns.set(rc = {"figure.dpi":300})
+sns.set(rc = {"figure.figsize":(6,3)})
+sns.set(font_scale = 0.5)
+
+
+def get_cloudmask_frame(df:pd.DataFrame, column:str) -> pd.DataFrame:
+    """ Create a data frame which is easy to plot with the number of observations 
+        for the unclouded or no information state for a given column of the data frame.
+
+    Args:
+        df (pd.DataFrame): Data frame that contains the CLM and at least another column to count the entries.
+        column (str): The column to be counted.
+
+    Returns:
+        pd.DataFrame: Data frame that is ready to plot.
+    """
+    # create two series with the number of observations that are unclouded or have no CLM information 
+    unclouded = df[df['CLM']==0].groupby(column)[column] \
+    .count()
+    noinfo = df[df['CLM']==255].groupby(column)[column] \
+    .count()
+    # create a data frame of the two series
+    cloudmask_df = pd.concat([unclouded, noinfo], axis=1)
+    cloudmask_df.columns = ['unclouded', 'no information']
+    cloudmask_df = cloudmask_df.reset_index()
+    # melts the data frame in order to make it plot-able
+    cloudmask_df = cloudmask_df.melt(id_vars=column).rename(columns=str.title)
+    cloudmask_df.columns = [column.title(), 'CLM', 'Count']
+
+    return cloudmask_df
+
+
+def get_info_per_field_id(df:pd.DataFrame, column:str) -> pd.DataFrame:
```

Comment:
Change this function to get_count_per_field_id

Reply:
I changed the name. 👍